test/integration/gardenlet/seed/seed/seed_suite_test.go

```diff
@@ -171,7 +175,8 @@ var _ = BeforeSuite(func() {
 		Config: config.GardenletConfiguration{
 			Controllers: &config.GardenletControllerConfiguration{
 				Seed: &config.SeedControllerConfiguration{
-					SyncPeriod: &metav1.Duration{Duration: time.Minute},
+					// This controller is pretty heavy-weight, so use DefaultControllerSyncPeriod
+					SyncPeriod: &gardenletconfigv1alpha1.DefaultControllerSyncPeriod,
```

Comment:
Let's rather keep it like before (not using this variable but `time.Minute` directly).